hummingbot/core/data_type/in_flight_order.py

```diff
@@ -32,8 +32,8 @@ class OrderState(Enum):
 
 class OrderUpdate(NamedTuple):
     trading_pair: str
-    update_timestamp: float  # seconds
     new_state: OrderState
+    update_timestamp: Optional[float] = None  # seconds
```

Comment:
In my opinion we want to have the update timestamp every time we process an update